sonar-php-plugin/src/main/java/org/sonar/plugins/php/reports/phpunit/PhpUnitReportImporter.java

```diff
@@ -55,4 +55,10 @@ public List<File> getReportFiles(SensorContext context) {
     return ExternalReportProvider.getReportFiles(context, reportPathKey());
   }
 
+  protected void formatAndCreateWarning(String warningFormat, File coverageReportFile) {
```

Comment:
I find it not so relevant to give implementation-details in the name of the method here. For the usage of this method, all that matters is that we will create a warning, so I would change the signature to
```suggestion
  protected void createWarning(String message, File file) {
```